src/RelayProvider.ts

```diff
@@ -366,7 +366,8 @@ export default class RelayProvider implements HttpProvider {
 
     _ethSendTransaction(
         payload: JsonRpcPayload,
-        callback: JsonRpcCallback
+        callback: JsonRpcCallback,
+        nonBlock?: boolean
```

Comment:
I don't think we should add this parameter here because it may be ignored when this method is called by the `send` method. The `RelayProvider` implements the `HttpProvider` that extends the `HttpProviderBase`. There is a `send` method with the following signature:
```typescript
 send(
        payload: JsonRpcPayload,
        callback: (error: Error | null, result?: JsonRpcResponse) => void
    ): void;
``` 

This method is overwritten by the RelayProvider, so I think we may need to add the `nonBlock` parameter in the payload directly to be compatible with the interface. Furthermore, we may need to change the [SDK](https://github.com/rsksmart/rif-relay-sdk/blob/master/src/sdk.ts#L302) as well to receive this parameter and/or to include it in the payload of the request.

**NIT**: I would change the name of the parameter to `waitForTransactionReceipt`, just to clarify what it's used for.

Reply:
I just did the changes :)